README.md

```diff
@@ -284,9 +279,9 @@ Once the workflow completes successfully, all documents will be imported into th
    - On the console page, you'll find an input bar. Enter your questions or queries related to the documents you've uploaded.
    - The app will provide summarized answers based on the content of your documents, along with references to the specific source documents.
 
-### Search and Explore from EKS Web UI
+### Search and Explore from EKS Web UI (Skip this if you have not configured the access to the Web UI Search Application)
```

Comment:
Nit: it's odd to give full sentence directions in a section header, how about the following...

Instead of 
```
### Search and Explore from EKS Web UI (Skip this if you have not configured the access to the Web UI Search Application)
```

I suggest:
```
### Search and Explore from EKS Web UI
_Note: This section only applies if you completed the "configure access to the Web UI search application" step under the deployment guide. 
```

Reply:
Yes, I will make this change.